### DISCUSSION TITLE
Loco positioning deck connection issue

### DISCUSSION THREAD

**Discussion by AL227git** [bitcraze/discussions#1658] [Q&A] [answered]
Hi, I'm trying to set up the Loco Positioning System, but I'm having some trouble. I followed the setup process outlined in the documentation https://www.bitcraze.io/documentation/tutorials/getting-started-with-loco-positioning-system/. 

I'm using 4 anchors: 2 at the bottom and 2 at the top. The anchors don't seem to appear on the Crazyflie client interface, even after several reconfigurations in TWR mode. For all 4 nodes, the RX and SFD lights flicker, while the TX and RXOK lights do not. After attempting to configure to TDoA mode and encountering the same issue without success, I realized that none of the LEDs on the Loco Positioning Deck are flashing. 

However, when testing the conductivity of the long pin headers installed on the Crazyflie 2.1, the connections appear normal. This leads me to believe that the positioning deck is not functioning properly and is therefore not relaying information from the nodes to the Crazyradio 2.0 for connection. 

What steps can I take to verify the connectivity between the anchors and the Crazyradio 2.0? Furthermore, how can I be certain that the positioning deck is not functioning as expected and may need to be returned for replacement?

**Comment by Bananazooka** [comment, marked as answer]
Hi, have you checked that your deck is facing the right direction? You can check [here](https://www.bitcraze.io/documentation/tutorials/getting-started-with-expansion-decks/)

**Reply by AL227git** [reply]
Thanks for the feedback!  (The deck was right side up but backwards.)
Problem solved.
